imageio/core/format.py

```diff
@@ -631,7 +631,18 @@ class FormatManager(object):
 
     @property
     def _formats(self):
-        return [x for x in known_plugins.values() if x.is_legacy]
+        available_formats = list()
```

Comment:
```suggestion
        available_formats = []
```

This is faster and recommended